### DISCUSSION TITLE
How to use 2 fields from same reference model?

### DISCUSSION THREAD

**Discussion by chutchapon** [prisma/prisma#10480] [Q&A] [answered]
How do I use 2 fields from same reference model eg. like 
```prisma
model Post {
  id             Int          @id @default(autoincrement())
  thumbnail      String       @default("https://via.placeholder.com/512x512")
  title          String
  content        String
  createdAt      DateTime     @default(now())
  updatedAt      DateTime     @updatedAt
  author         User         @relation(fields: [authorId], references: [id])
  authorId       Int
  language       Language     @relation(fields: [languageId], references: [id])
  languageId     Int
  category       PostCategory @relation(fields: [postCategoryId], references: [id])
  tags           TagOnPosts[]
  postCategoryId Int
  attachments    Attachment[] // <== error here
  albums         Attachment[] // 
}

model Attachment {
  id        Int      @id @default(autoincrement())
  filename  String
  mimetype  String
  encoding  String
  path      String
  size      Int
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
  Post      Post?    @relation(fields: [postId], references: [id])
  postId    Int?
}

``
Thanks for advance.

**Comment by TasinIshmam** [comment, marked as answer]
Hey @chutchapon :wave:  

If you have multiple relations between the same models, you will need to disambiguate them by giving names to each relation. Here's [more information](https://www.prisma.io/docs/concepts/components/prisma-schema/relations#disambiguating-relations) on the docs about this.

**Reply by chutchapon** [reply]
Thank you for the information

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
